internal/cachecontroller/cache_controller.go

```diff
@@ -108,7 +117,7 @@ func (c *InMemoryCacheController) findChanges(ctx context.Context, storeID strin
 	return c.ds.ReadChanges(ctx, storeID, storage.ReadChangesFilter{}, opts)
 }
 
-func (c *InMemoryCacheController) findChangesAndInvalidate(ctx context.Context, storeID string) (time.Time, error) {
+func (c *InMemoryCacheController) findChangesAndInvalidate(ctx context.Context, storeID string) error {
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(attribute.Bool("invalidations", true))
```

Comment:
This is no longer going to be visible. It should generate it's own span and be linked to the parent.